xml/operations-troubleshooting-recover_rabbit.xml

```diff
@@ -204,29 +204,68 @@ ardana-cp1-c1-m3-mgmt</screen>
    baremetal node, the &rabbit; cluster needs any references to the node
    removed from it. This is because &rabbit; associates a cookie with each node
    in the cluster which is derived, in part, by the specific hardware.
-   Replacing a hard drive in a node should be fine, but an exchanged
-   motherboard or replacing it with another node entirely will potentially
-   trigger a breakage. Under these circumstances, the running &rabbit; cluster
-   must be edited from a running &rabbit; node with these steps:
+   Replacing a hard drive in a node should be fine, but changing a motherboard
```

Comment:
In general, we avoid using _should_ and _would_ in technical documentation as this introduces ambiguity. It's either fine or it's not. Also, _trigger a breakage_ sounds clunky. Maybe something like this?

_While it's possible to replace a hard drive without problems, swapping a motherboard may cause interruption._

Reply:
Thank you for proofreading and copy editing. Fixed and then some. The whole paragraph was clunky.